### PR TITLE
`<IconButton/>`- Adding tiny and large styling & updating documentation

### PR DESCRIPTION
**Pull request by TalZa1810**

### CODE REVIEW

File: packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css [hunk 1]
```diff
@@ -193,19 +193,30 @@
...
+  height: 24px;
+}
+
 .iconButton.small {
   width: 30px;
   height: 30px;
 }
 
+.iconButton.large {
+  width: 49px;
```
Comment (shlomitc) [inline packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css]: why 49?

Comment (TalZa1810) [reply packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css]: According to Zeplin:
lll

Comment (shlomitc) [reply packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css]: doesn't seem right - @LittleTal ?

Comment (LittleTal) [reply packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css]: @shlomitc @TalZa1810 It's a mistake, should be 48

Comment (TalZa1810) [reply packages/wix-ui-core/src/themes/backoffice/icon-button/icon-button.st.css]: @shlomitc @LittleTal 
Fixed

**Comment by TalZa1810** [comment]
Hey @mosheKerbel ,
I need the styles in wix-ui-core in order to resolve this issue:
https://github.com/wix/wix-style-react/issues/3431 

I've implemented both tiny & large and added the examples under the title:
`Sizes - Icon Buttons (sizes)`:
https://wix-wix-ui-core-pr-1326.surge.sh/?path=/story/backoffice--buttons.


Thanks :)

**Comment by mosheKerbel** [comment]
@TalZa1810 looks like tests are flaky, @jonathanadler suggested to create some dummy commit in order to re-trigger the build.

**Pull request merged**